xstatic/pkg/datatables/__init__.py

```diff
@@ -46,7 +46,7 @@
     # your relative path/filenames should usually be without version
     # information, because either the base dir/url is exactly for this
     # version or the mapping will care for accessing this version.
-    ('microsoft', 'http'): 'http://ajax.aspnetcdn.com/ajax/jquery.dataTables/%s' % VERSION,
-    ('microsoft', 'https'): 'https://ajax.aspnetcdn.com/ajax/jquery.dataTables/%s' % VERSION,
+    ('datatables', 'css'): 'https://cdn.datatables.net/%s/css/jquery.dataTables.min.css' % VERSION,
+    ('datatables', 'js'): 'https://cdn.datatables.net/%s/js/jquery.dataTables.min.js' % VERSION,
```

Comment:
I'm not super positive about these two lines ? The xstatic libs I've worked with were not available over CDN.